examples/docs/src/pages/themes/docs/configuration.mdx

```diff
@@ -0,0 +1,618 @@
+
+**Type:** `string`
+
+**Default:** `https://github.com/shuding/nextra`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  docsRepositoryBase: 'https://github.com/shuding/nextra'
+}
+```
+
+## `titleSuffix`
+
+Suffix that will be added to page titles in the URL bar.
+
+**Type:** `string`
+
+**Default:** `– Nextra`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  titleSuffix: '– Nextra'
+}
+```
+
+## `nextLinks` and `prevLinks`
+
+Specifies if arrows are being shown at the bottom
+of a page showing the next and previous page, like
+the ones at the bottom of this page.
+
+**Type:** `boolean`
+
+**Default:** `true`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  nextLinks: true,
+  prevLinks: true
+}
+```
+
+## `search`
+
+Specifies if a search box should be shown in the top right.
+
+**Type:** `boolean`
+
+**Default:** `true`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  search: true
+}
+```
+
+## `searchPlaceholder`
+
+Specifies if a search box should be shown in the top right.
+
+**Type:** `string | ((props: { locale?: string }) => string){:ts}`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  searchPlaceholder: ({ locale }) => {
+    if (locale === 'zh-CN') return '搜索文档...'
+    return 'Search documentation...'
+  }
+}
+```
+
+## `customSearch`
+
+A custom component to display instead of the search bar in the top right, for example Algolia.
+
+**Type:** `ReactNode`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+import Search from 'your-search'
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  customSearch: <Search />
+}
+```
+
+## `darkMode`
+
+Specifies if the user can select a dark mode.
+
+**Type:** `boolean`
+
+**Default:** `true`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  darkMode: true
+}
+```
+
+## `defaultMenuCollapsed`
+
+Specifies if the menu on the left is collapsed by default.
+
+**Type:** `boolean`
+
+**Default:** `false`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  defaultMenuCollapsed: false
+}
+```
+
+## `font`
+
+Specifies if nextra should load its own fonts. Disable this if you want to use a custom font.
+
+**Type:** `boolean`
+
+**Default:** `true`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  font: true
+}
+```
```

Comment:
currently, there is no logic that handles `fonts`, so this part can be commented